br/pkg/lightning/backend/external/merge.go

```diff
@@ -60,6 +63,7 @@ func MergeOverlappingFiles(ctx context.Context, paths []string, store storage.Ex
 	return eg.Wait()
 }
 
+// unused for now.
```

Comment:
since it's not used, no need to change line 88~123?

Reply:
Github show the difference, but I didn't change them